action.yml

```diff
@@ -281,3 +279,35 @@ runs:
         phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml
         ./codecov-linux -f coverage.xml;
         echo "coverage.xml generated and uploaded to codecov"
+
+    - name: "Run documentation linting"
+      if: ${{ inputs.doclinting == 'true' }}
+      shell: bash
+      run: |
+        echo "Running documentation linting"
+        if [[ ! -f vendor/silverstripe/documentation-lint/.nvmrc ]]; then
+          echo "File vendor/silverstripe/documentation-lint/.nvmrc is missing. Check dependencies."
+          exit 1
+        fi
+        # Remove any sneaky attempts to put __install-nvm.sh into pull-requests
+        if [[ -f __install-nvm.sh ]]; then
+          rm __install-nvm.sh
+        fi
+        # Install nvm
+        cp ${{ github.action_path }}/install-nvm.sh __install-nvm.sh
+        chmod +x __install-nvm.sh
+        ./__install-nvm.sh
+        if [[ $? != 0 ]]; then
+          echo "Error while installing nvm"
+          exit 1
+        fi
+        export NVM_DIR="$HOME/.nvm"
+        # this loads nvm into the current terminal
+        [[ -s "$NVM_DIR/nvm.sh" ]] && \. "$NVM_DIR/nvm.sh"
+        # Swap to correct version and make sure yarn is installed
+        NPM_VERSION=$(cat vendor/silverstripe/documentation-lint/.nvmrc)
+        nvm install $NPM_VERSION && nvm use $NPM_VERSION
+        npm install --global yarn
+        # Run the linting script
+        vendor/bin/ss-docs-lint
```

Comment:
```suggestion
        vendor/bin/doclint
```

Reply:
I don't like this but I'll do it to avoid unnecessary ping pong. We can always change it later if we find a reason to.

Reply:
Done